tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
FYI @keithamus we updated the [polyfill repo template](https://github.com/oddbird/polyfill-template) but haven't yet pulled in all the new changes into this repo. There are quite a number of new things and changes so I'll let you decide if you would like to pull those in or if it would be too much. Feel free to use the template repo as a reference and pull things as you need.

Reply:
I’ll update this tomorrow to include the changes from the template!